app/build.gradle

```diff
@@ -214,6 +214,7 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation "androidx.room:room-runtime:${androidxRoomVersion}"
     implementation "androidx.room:room-rxjava3:${androidxRoomVersion}"
+    kapt "org.xerial:sqlite-jdbc:3.34.0"
```

Comment:
Does this have an impact on apk size?

Reply:
Can you please help me check? It won't build on a M1 Mac without it: https://issuetracker.google.com/issues/174695268#comment9

Reply:
Nevermind, I used GitHub Actions.

```
15831094 14 Aug 08:39 app-debug-xerial.apk
15830642 14 Aug 08:36 app-debug.apk
```

So almost the same size.

https://github.com/ktprograms/NewPipe/actions

Reply:
Can you add a comment that the dependency is only as a workaround for M1 Max and link the thread of the underlying issue. So that when the underlying issue is solved, the dependecy can be removed.

Reply:
Fixed in https://github.com/TeamNewPipe/NewPipe/pull/6919/commits/65f2730261ae1c0cc544223caee666c4060d4f71